stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-16.24
+resolver: lts-18.5
```

Comment:
Looks like this isn't quite sufficient to get the stack CI build to work. I've not been using stack much lately, do you think you could look into that failure?

Reply:
Sorry, I should've done this in another PR. I just wanted to avoid building an old snapshot on my local computer when working on this. But yeah, I'll get this working.